bigframes/dataframe.py

```diff
@@ -1305,6 +1305,36 @@ def nsmallest(
         column_ids = self._sql_names(columns)
         return DataFrame(block_ops.nsmallest(self._block, n, column_ids, keep=keep))
 
+    def insert(
+        self,
+        loc: int,
+        column: blocks.Label,
+        value: SingleItemValue,
+        allow_duplicates: bool = False,
+    ):
+        column_count = len(self.columns)
+        if loc > column_count:
+            raise IndexError(
+                f"index {loc} is out of bounds for axis 0 with size {column_count}"
```

Comment:
axis 0 doesn't exist in this context

Reply:
Yes, but this is the message pandas used for insert for some reason. Maybe just mean the columns.

Reply:
Updated.